content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -231,6 +235,11 @@ Update a FamilyMemberHistory.
 
     PUT /FamilyMemberHistory/:id
 
+_Implementation Notes_
+
+* Conditions can be removed from a FamilyMemberHistory by setting the 'condition-lifecycle-status' modifierExtension to 'entered-in-error'. Only existing conditions can be removed; condition id must be provided.
+* An update should only be preformed directly after a FamilyMemberHistory is retrieved. If a condition.id is provided from the get operation, it MUST be provided on the subsequent update and must match the condition.id returned from the get.
+
 ### Authorization Types
 
 <%= authorization_types(provider: true, system: true, patient: false) %>
```

Comment:
Should we mention that if you're trying to change the status of a FMH to health-unknown or completed with no conditions.... that u must ensure all conditions are removed first 
I think im thinking about that right? they will again get a generic 400 if they dont remove the conditions first

Reply:
Hmm...let's leave it off for now. I think we'll need to come up with a good way of explaining the workflows with the specific values, which we can add a little later.